src/Sylius/Behat/Context/Api/Admin/ManagingProductReviewsContext.php

```diff
@@ -0,0 +1,237 @@
+    }
+
+    /**
+     * @When I change its title to :title
+     */
+    public function iChangeItsTitleTo(string $title): void
+    {
+        $this->client->addRequestData('title', $title);
+    }
+
+    /**
+     * @When I change its comment to :comment
+     */
+    public function iChangeItsCommentTo(string $comment): void
+    {
+        $this->client->addRequestData('comment', $comment);
+    }
+
+    /**
+     * @When /^I remove its (comment|title)$/
+     */
+    public function iRemoveItsComment(string $element): void
+    {
+        $this->client->addRequestData($element, '');
+    }
+
+    /**
+     * @When I save my changes
+     * @When I try to save my changes
+     */
+    public function iSaveMyChanges(): void
+    {
+        $this->client->update();
+    }
+
+    /**
+     * @Then /^(this product review) (comment|title) should be "([^"]+)"$/
+     */
+    public function thisProductReviewElementShouldBeValue(ReviewInterface $productReview, string $element, string $value): void
+    {
+        $this->assertIfReviewHasElementWithValue($productReview, $element, $value);
+    }
+
+    /**
+     * @Then /^(this product review) rating should be (\d+)$/
+     */
+    public function thisProductReviewRatingShouldBe(ReviewInterface $productReview, int $rating): void
+    {
+        $this->assertIfReviewHasElementWithValue($productReview, 'rating', $rating);
+    }
+
+    /**
+     * @When I choose :rating as its rating
+     */
+    public function iChooseAsItsRating(int $rating): void
+    {
+        $this->client->addRequestData('rating', $rating);
+    }
+
+    /**
+     * @When I accept the :productReview product review
+     */
+    public function iAcceptTheProductReview(ReviewInterface $productReview): void
+    {
+        $this->client->applyTransition('product_reviews',(string) $productReview->getId(), ProductReviewTransitions::TRANSITION_ACCEPT);
+    }
+
+    /**
+     * @When I reject the :productReview product review
+     */
+    public function iRejectTheProductReview(ReviewInterface $productReview): void
+    {
+        $this->client->applyTransition('product_reviews', (string) $productReview->getId(), ProductReviewTransitions::TRANSITION_REJECT);
+    }
+
+    /**
+     * @Then /^(this product review) status should be "([^"]+)"$/
+     */
+    public function thisProductReviewStatusShouldBe(ReviewInterface $productReview, string $status): void
+    {
+        $this->assertIfReviewHasElementWithValue($productReview, 'status', $status);
+    }
+
+    /**
+     * @Then /^I should be notified that it has been successfully (accepted|rejected)$/
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyUpdated(string $action): void
+    {
+        $this->assertIfReviewHasElementWithValue($this->sharedStorage->get('product_review'), 'status', $action);
+    }
+
+    /**
+     * @When I delete the :productReview product review
```

Comment:
Please, reorder steps implementations to have them in `Given` -> `When` -> `Then` order 😄